webrtc/videoroom_advanced/lib/videoroom/application.ex

```diff
@@ -16,6 +21,45 @@ defmodule VideoRoom.Application do
     Supervisor.start_link(children, opts)
   end
 
+  @impl true
+  def stop(_state) do
+    delete_cert_file()
+    :ok
+  end
+
+  @spec get_cert_file_path() :: binary()
+  def get_cert_file_path(), do: @cert_file_path
+
+  defp create_turn_cert_file() do
+    try do
+      cert_path = Application.fetch_env!(:membrane_videoroom_demo, :tls_turn_cert)
+      pkey_path = Application.fetch_env!(:membrane_videoroom_demo, :tls_turn_pkey)
+
+      if cert_path != nil and pkey_path != nil do
+        cert = File.read!(cert_path)
+        pkey = File.read!(pkey_path)
+
+        File.touch!(@cert_file_path)
+        File.chmod!(@cert_file_path, 0o600)
+        File.write!(@cert_file_path, "#{cert}\n#{pkey}")
+
+        Application.put_env(:membrane_videoroom_demo, :integrated_turn_certfile, @cert_file_path)
+      else
+        Membrane.Logger.info(
+          "Path to TLS cert or private key was not provided. TLS TURN will not run"
+        )
+      end
+    rescue
+      File.Error ->
+        Membrane.Logger.error(
+          "Error occured while reading certificates for TLS TURN, so it will not run. \
+          Ensure, that paths to TLS certificate and private key are correct."
+        )
+    end
```

Comment:
You can write something like this to avoid using `try rescue`
```suggestion
      cert_path = Application.fetch_env(:membrane_videoroom_demo, :tls_turn_cert)
      pkey_path = Application.fetch_env(:membrane_videoroom_demo, :tls_turn_pkey)
      
      if cert_path == :error or pkey_path == :error do
        Membrane.Logger.warning("""
          Integrated TURN certificate or private key path not specified.
          Integrated TURN will not handle TLS connections.
          """ )
      else 
        cert = File.read!(cert_path)
        pkey = File.read!(pkey_path)

        File.touch!(@cert_file_path)
        File.chmod!(@cert_file_path, 0o600)
        File.write!(@cert_file_path, "#{cert}\n#{pkey}")

        Application.put_env(:membrane_videoroom_demo, :integrated_turn_certfile, @cert_file_path)
      end
```